### PR TITLE
monitor-a-dm-cluster: add more metrics

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/756
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/monitor-a-dm-cluster.md [hunk 1]
```diff
@@ -78,7 +79,8 @@ The following metrics show only when `task-mode` is in the `incremental` or `all
 | Metric name | Description | Alert | Severity level |
 |:----|:------------|:----|:----|
 | remaining time to sync | The predicted remaining time it takes `syncer` to be completely migrated with the master (in minutes) | N/A | N/A |
-| replicate lag | The latency time it takes to replicate the binlog from master to `syncer` (in seconds) | N/A | N/A |
+| replicate lag gauge | The latency time it takes to replicate the binlog from master to `syncer` (in seconds) | N/A | N/A |
+| replicate lag histogram | The histogram of replicating the binlog from master to `syncer` (in seconds). Note that due to different statistical mechanisms, errors exist in the data | N/A | N/A |
```
Comment (Ehco1996) [inline en/monitor-a-dm-cluster.md:83]: maybe use Inaccuracy instead of errors here is better ?

**Review by Ehco1996 — COMMENTED** [review]
rest LGTM

File: en/monitor-a-dm-cluster.md (hunk 1 above)
Comment (Liuxiaozhen12) [inline en/monitor-a-dm-cluster.md:83]:
```suggestion
| replicate lag histogram | The histogram of replicating the binlog from master to `syncer` (in seconds). Note that due to different statistical mechanisms, the data might be inaccurate | N/A | N/A |
```

Comment (Liuxiaozhen12) [reply en/monitor-a-dm-cluster.md]: Yes you're right~ Error usually means an error message in computing. Fixed in https://github.com/pingcap/docs-dm/pull/768/commits/443a1023aec8af1b12bfe17bf5de253389e24535. Thx!

**Review by Ehco1996 — APPROVED** [review]

File: en/monitor-a-dm-cluster.md [hunk 2]
```diff
@@ -78,7 +79,8 @@ The following metrics show only when `task-mode` is in the `incremental` or `all
 | Metric name | Description | Alert | Severity level |
 |:----|:------------|:----|:----|
 | remaining time to sync | The predicted remaining time it takes `syncer` to be completely migrated with the master (in minutes) | N/A | N/A |
-| replicate lag | The latency time it takes to replicate the binlog from master to `syncer` (in seconds) | N/A | N/A |
+| replicate lag gauge | The latency time it takes to replicate the binlog from master to `syncer` (in seconds) | N/A | N/A |
```
Comment (lance6716) [inline en/monitor-a-dm-cluster.md:82]:
```suggestion
| replicate lag gauge | The latency time it takes to replicate the binlog from upstream to downstream` (in seconds) | N/A | N/A |
```

Comment (lance6716) [reply en/monitor-a-dm-cluster.md:82]: seems other old description is not right ...

Comment (Liuxiaozhen12) [reply en/monitor-a-dm-cluster.md]: fixed in https://github.com/pingcap/docs-dm/pull/768/commits/8153636bdfd9e75c5b20a46ed439a1a1564eb995。 PTAL~

**Comment by Liuxiaozhen12** [comment]
@lance6716 fixed in https://github.com/pingcap/docs-dm/pull/768/commits/0da70dd7713d47b5ad936eb51d2ae45d878bee48, PTAL. I'm not sure where I should mention syncer and where to mention the downstream.

**Comment by lance6716** [comment]
lgtm

**Review by TomShawn — APPROVED** [review]

**Pull request merged**